fast_td3/fast_td3/environments/humanoid_bench_env_v2.py

```
@@ -0,0 +1,129 @@
+        # This will be used for getting 'true' next observations
+        infos = dict()
+        infos["observations"] = {"raw": {"obs": observations.copy()}}
+        truncateds = np.zeros_like(dones)
+        for i in range(self.num_envs):
+            if raw_infos[i].get("TimeLimit.truncated", False):
+                truncateds[i] = True
+                infos["observations"]["raw"]["obs"][i] = raw_infos[i][
+                    "terminal_observation"
+                ]
+
+        observations_flat = np.concatenate([observations["pelvis_orientation"], observations["pelvis_velocites"],observations["joint_positions"], observations["joint_velocities"]], axis=-1)
```

Ruff (0.14.5)

Local variable `observations_flat` is assigned to but never used

Remove assignment to unused variable `observations_flat`

(F841)